automation/terraform/monitoring/README.md

```diff
@@ -0,0 +1,180 @@
+
+**Note:** all manual steps should be run from the [automation monitoring](https://github.com/MinaProtocol/mina/tree/develop/automation/terraform/monitoring) directory within the *mina* repo. A copy of the rendered rules configuration will be placed in this directory for reference when testing locally.
+
+##### Linting
+
+[lints](https://github.com/grafana/cortex-tools#rules-lint) a testnet alert rules file. The linter's aim is not to verify correctness but just YAML and PromQL expression formatting within the rule file.
+
+###### automation
+
+Executed by CI's *Lint/TestnetAlerts* [job](https://github.com/MinaProtocol/mina/blob/develop/buildkite/src/Jobs/Lint/TestnetAlerts.dhall) when a change is detected to the testnet-alerts template file.
+
+###### manual steps
+
+```
+    terraform apply -target module.o1testnet_alerts.null_resource.alert_rules_lint
+``` 
+
+##### Check alerts against recommended [best practices](https://prometheus.io/docs/practices/rules/)
+
+###### automation
+
+Executed by CI's *Lint/TestnetAlerts* [job](https://github.com/MinaProtocol/mina/blob/develop/buildkite/src/Jobs/Lint/TestnetAlerts.dhall) when a change is detected to the testnet-alerts template file.
+
+###### manual steps
+
+```
+    terraform apply -target module.o1testnet_alerts.null_resource.alert_rules_check
+```
+
+#### Deployment
+
+Deploying testnet alert rules entails syncing the rendered configuration in source with the Prometheus instance rules config found [here](https://o1testnet.grafana.net/a/grafana-alerting-ui-app/?tab=rules&rulessource=grafanacloud-o1testnet-prom). Appropriate AWS access is necessary for authenticating with Grafanacloud and should be similar to, if not the same, as those used for deploying testnets.
+
+###### automation
+
+Executed by CI's *Release/TestnetAlerts* [job](https://github.com/MinaProtocol/mina/blob/develop/buildkite/src/Jobs/Release/TestnetAlerts.dhall) when a change is detected to the testnet-alerts template file and linting/checking of alerts has succeeded.
+
+###### manual steps
+
+```
+    terraform apply -target module.o1testnet_alerts.docker_container.sync_alert_rules
+```
+
+**Note:** operation will sync provisioned alerts with exact match of alert file state (e.g. alerts removed from the alert file will be unprovisioned on Grafanacloud)
+
+**Warning:** Unfortunately terraform silences errors which cortextool is producing, for example when syncing rules. In order to debug sync rules process:
+- Obtain secrets (id,address,key) from aws secret manager 
+- Run above terraform command once again
+- Run below command in automation/terraform/monitoring folder: 
```

Comment:
```suggestion
    - Obtain secrets (id,address,key) from AWS secret manager 
    - Run the previous terraform command once again
    - Run the following command in automation/terraform/monitoring folder: 
```

Reply:
instead of above, use earlier or preceding (on in this case, previous), see https://developers.google.com/style/word-list#above 

instead of below, use later or following, see https://developers.google.com/style/word-list#below

the indentation of the list items indents the steps for the process